index.js

```diff
@@ -48,11 +49,9 @@ function processRules(rule, params) {
     if (params.indexName) vals[params.indexName] = i;
 
     rule.nodes.forEach(node => {
-      const clone = node.clone();
-      const proxy = postcss.rule({ nodes: [clone] });
-
```

Comment:
It appears postcss 8 will clone nodes for you: https://github.com/postcss/postcss/blob/d41a37605adc16b3eb17f806ce0259a8b316e603/lib/node.js#L42-L46